roles/pulp_devel/tasks/galaxy_ui.yml

```diff
@@ -67,6 +58,8 @@
 
 - name: Download galaxy ui from github
   command:
-    cmd: python setup.py prepare_static --force-download-ui
```

Comment:
Isn't this more robust? For say Centos 7 where system python is Python 2?

Reply:
it was breaking the CI before